crates/language/Cargo.toml

```diff
@@ -20,6 +20,7 @@ tree-sitter-bash = { version = "0.20.5", optional = true }
 tree-sitter-c = { version = "0.20.8", optional = true }
 tree-sitter-cpp = { version = "0.20.5", optional = true }
 tree-sitter-c-sharp = { version = "0.20.0", package = "ast-grep-tree-sitter-c-sharp", optional = true }
+tree-sitter-cmake = { version = "=0.4.1", optional = true }
```

Comment:
This is needed because 0.4.2 brings in tree-sitter 0.21, and that breaks the build at the moment, due to mismatched TSLanguage types.